ui/StatusQ/src/StatusQ/Components/StatusVideo.qml

```diff
@@ -0,0 +1,29 @@
+import QtQuick 2.13
+import QtGraphicalEffects 1.0
+import QtMultimedia 5.15
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+
+Rectangle {
```

Comment:
Same suggestion than before. What about using `Control` as a base class?

Reply:
This looks like a non-interactive control so `Item`-based root controls look more suitable. I prefer spacing using layouts and anchors to `Control`'s way of padding and inserts for internal/delegate items.

Looking at the Control's super-classes in Qt none looks very similar in requirements to this `StatusVideo`: `AbstractButton, BusyIndicator, ComboBox, Container, Dial, MenuSeparator, PageIndicator, ProgressBar, RangeSlider, ScrollBar, ScrollIndicator, Slider, SpinBox, StackView, ToolSeparator, Tumbler`
This looks more similar to QML's `Image` which is `Item`-based (as `Rectangle`).